config/types.go

```diff
@@ -180,21 +180,26 @@ type ServiceConfig struct {
 }
 
 type RancherConfig struct {
-	LbConfig                 *LBConfig                        `yaml:"lb_config"`
+	LbConfig                 *LBConfig                        `yaml:"lb_config,omitempty"`
 	LegacyLoadBalancerConfig *legacyClient.LoadBalancerConfig `yaml:"load_balancer_config,omitempty"`
 	DefaultCert              string                           `yaml:"default_cert,omitempty"`
 	Certs                    []string                         `yaml:"certs,omitempty"`
 
-	Vcpu     yaml.StringorInt            `yaml:"vcpu,omitempty"`
-	Userdata string                      `yaml:"userdata,omitempty"`
-	Memory   yaml.MemStringorInt         `yaml:"memory,omitempty"`
-	Disks    []client.VirtualMachineDisk `yaml:"disks,omitempty"`
+	Vcpu                yaml.StringorInt            `yaml:"vcpu,omitempty"`
+	Userdata            string                      `yaml:"userdata,omitempty"`
+	Memory              yaml.MemStringorInt         `yaml:"memory,omitempty"`
+	Disks               []client.VirtualMachineDisk `yaml:"disks,omitempty"`
```

Comment:
These four lines are grouped together because they are VM fields.  You can put a comment indicating that.  But the more important part is MilliCpuReservation shouldn't be in this group but the next group